data/kernels/colorspace.h

```diff
@@ -624,13 +624,13 @@ static inline float4 dt_uvY_to_xyY(const float4 uvY)
 
 static inline float4 dt_XYZ_to_xyY(const float4 XYZ)
 {
-  // see cpu implementation for details
+  // see cpu implementation for details, use D65_xy as fallback
   float4 xyY;
   const float sum = XYZ.x + XYZ.y + XYZ.z;
   if(XYZ.x == 0.0f && XYZ.y == 0.0f && XYZ.z == 0.0f)
   {
-    xyY.x = 2.0f / 4.5f;
-    xyY.y = 1.0f / 4.5f;
+    xyY.x = 0.31269999999999992f;
+    xyY.y = 0.32899999999999996f;
```

Comment:
I suspect these values are really 0.3127 and 0.3290, forced to the nearest representable value as a float or double.  There's a variable `WhitePoint` set to those values in iop/temperature.c.

Reply:
I took those from wiki

Reply:
In colorspaces.c:

```
//D65 (sRGB, AdobeRGB, Rec2020)
static const cmsCIExyY D65xyY = {0.312700492, 0.329000939, 1.0};
```

We probably want to move this into the header file and use the values here.

Reply:
You are both right ... fixed according to your suggestions, squashed and force-pushed